app/main.py

```diff
@@ -1,3 +1,39 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.data_handler import (add_models,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line